packages/button/src/LionButton.js

```diff
@@ -111,6 +110,22 @@ export class LionButton extends DisabledWithTabIndexMixin(SlotMixin(LitElement))
           fill: #adadad;
         }
       `,
+      ...(this.__isIE11() /* visibility hidden in IE11 breaks native button functionality */
+        ? [
+            /* TODO: Make SR-only css mixin? */
+            css`
+              :host .btn ::slotted(button) {
+                clip: rect(0, 0, 0, 0);
```

Comment:
Please expand this a little further following this pattern: https://www.scottohara.me/blog/2017/04/14/inclusively-hidden.html#hiding-content-visually

```suggestion
                clip: rect(0 0 0 0); 
                clip-path: inset(50%);
                height: 1px;
                overflow: hidden;
                position: absolute;
                white-space: nowrap; 
                width: 1px;
```

Reply:
Yeah to be honest, in retrospect, I think we should create an srOnlyMixin that returns this css :)